README.md

```diff
@@ -41,6 +41,9 @@ $ cd leonardo
 ```
 
 ### Run Leonardo unit tests
+
+Leonardo requires Java 8 due to a dependency on Java's DNS SPI functionality. This feature is removed in Java 9 and above.
```

Comment:
I think we should always assume scala apps are only reliable on java 8, which is the only targeted java version.

Reply:
you mean scala in general only targets java 8?

Reply:
Yea. 2.13’s official target is also java 8.